### DISCUSSION TITLE
Yarn not run in subfolder context after cd'ing to sub folder

### DISCUSSION THREAD

**Discussion by dschinkel** [community/community#26900] [Actions]
<p>I ran the following:</p>
<pre><code class="lang-auto">- run: cd src/server
- run: yarn build-server
</code></pre>
<p>but looks like my yarn script is being run in context of <code>root</code> still, not in context of being in <code>src/server</code> as I have an <code>src/server/package.json</code> where I want to run that command from</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Each <code>run</code> step gets its own shell, so changing directories doesn’t carry over. Use the <a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun"><code>working-directory</code> keyword</a> for the step instead:</p>
<pre><code class="lang-yaml">- run: yarn build-server
  working-directory: src/server
</code></pre>
